src/js/profile/blocks/indicator.js

```diff
@@ -31,4 +34,26 @@ export class Indicator extends ContentBlock {
 
         }
     }
+
+    orderChartData() {
+        let primaryGroup = this.indicator.metadata.primary_group;
+        let groupsOrder = this.indicator.metadata.groups.filter((g) => {
+            return g.name === primaryGroup
+        })[0];
+        let orderedGroups = [];
+
+        groupsOrder.subindicators.forEach((g) => {
+            let d = this.indicator.data.filter((x) => {
+                return x[primaryGroup] === g
+            })[0];
+
+            if(d !== undefined){
+                orderedGroups.push(d);
```

Comment:
After removing 0 above this line should have been changed to:
`orderedGroups.push(...d);`